modules/ROOT/pages/service-mesh-faqs.adoc

```diff
@@ -34,17 +30,17 @@ No, and this is not required. Apps deployed to Anypoint Runtime Fabric are alrea
 +
 Anypoint Service Mesh supports GKE (Google), AKS (Azure), and EKS (AWS)
 
-== Features Covered in GA Release
+// == Features Covered in GA Release
```

Comment:
Thanks. You will delete all this, right?

Reply:
Done! Thanks!